mirai-core-api/src/commonMain/kotlin/utils/BotConfiguration.kt

```diff
@@ -245,6 +245,16 @@ public open class BotConfiguration { // open for Java
          */
         ANDROID_WATCH,
 
+        /**
+         * iPad - 来自MiraiGo
+         */
+        IPAD,
+
+        /**
+         * MacOS - 来自MiraiGo
+         */
+        MACOS,
```

Comment:
```suggestion
        /**
         * iPad - 来自MiraiGo
         * 
         * @since 2.8
         */
        IPAD,

        /**
         * MacOS - 来自MiraiGo
         * 
         * @since 2.8
         */
        MACOS,
```